Update fixture to include adviser in a International Trade Team (#2987)
uktrade/data-hub-api: changelog/update_test_data_fixture.internal.md

> New adviser being added to the test fixtures now allows us to run a full end to end test around managing a lead ITA.

Reviewer: ```suggestion
A new adviser has been added to the test fixtures. This adviser will be used in frontend end-to-end tests for managing a lead ITA.
```

I would reccomend updating the changelog so it's a little more formal
